package.json

```diff
@@ -48,9 +51,7 @@
     "semantic-release": "^15.13.19",
     "shimmer": "^1.2.1",
     "webpack": "^4.35.0",
-    "webpack-cli": "^3.3.4",
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@babel/plugin-proposal-optional-chaining": "^7.9.0"
+    "webpack-cli": "^3.3.4"
```

Comment:
Why we need this ?

Reply:
It was there before, I just added `fs-extra` to the devDependency